Zeeguu Reader/Zeeguu Reader/ArticleListViewController.swift

```diff
@@ -0,0 +1,135 @@
+//
+//  MasterViewController.swift
+//  Zeeguu Reader
+//
+//  Created by Jorrit Oosterhof on 08-12-15.
+//  Copyright © 2015 Jorrit Oosterhof.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+import ZeeguuAPI
+
+class ArticleListViewController: UITableViewController {
+
+	var objects = [Article]()
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		self.tableView.estimatedRowHeight = 80
+		
+		self.title = "APP_TITLE".localized
+		self.navigationItem.title = "APP_TITLE".localized
+		
+		let addButton = UIBarButtonItem(barButtonSystemItem: .Add, target: self, action: "insertNewObject:")
+		self.navigationItem.rightBarButtonItem = addButton
+		
+		let logoutButton = UIBarButtonItem(title: "Logout", style: .Done, target: self, action: "logout:")
```

Comment:
The button was just there to make it quick to logout, but as the button will be somewhere in the future, I fixed it